<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Instructions Dialog</string>
  </property>
  <widget class="QListWidget" name="instructions_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>741</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="instruction_text">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter instruction here...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_instruction">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>300</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_selected_instruction">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>340</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
